### PR TITLE
Create Notes-Up manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
Patches have been submitted upstream: https://github.com/Philip-Scott/Notes-up/pull/138

**Comment by Alexander-Wilms** [comment]
The author would like to maintain the repo: https://github.com/Philip-Scott

File: com.github.philip_scott.notes-up.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+	"app-id": "com.github.philip_scott.notes-up",
```
Comment (Alexander-Wilms) [inline com.github.philip_scott.notes-up.json]: replacing only the first dash with an underscore works, but shouldn't this actually be the other way around: https://github.com/flatpak/flatpak/issues/295#issuecomment-246260896 ?

**Comment by Alexander-Wilms** [comment]
@TingPing Could you trigger builds for this one as well?

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/buildrequests/2764?redirect_to_build=true

File: com.github.philip_scott.notes-up.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"cleanup": [
+		"/bin/granite-demo",
+		"/share/applications/granite-demo.desktop",
+		"/include",
+		"/lib/pkgconfig",
+		"/lib64/pkgconfig",
+		"/lib/debug",
```
Comment (TingPing) [inline com.github.philip_scott.notes-up.json]: Don't cleanup debug.

File: com.github.philip_scott.notes-up.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	"build-options": {
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"cleanup": [
+		"/bin/granite-demo",
+		"/share/applications/granite-demo.desktop",
+		"/include",
+		"/lib/pkgconfig",
+		"/lib64/pkgconfig",
```
Comment (TingPing) [inline com.github.philip_scott.notes-up.json]: Nothing should be installing into `/lib64` in the first place?

Comment (Alexander-Wilms) [inline com.github.philip_scott.notes-up.json]: Fixed

File: com.github.philip_scott.notes-up.json (hunk 2 above)
Comment (Alexander-Wilms) [inline com.github.philip_scott.notes-up.json]: Fixed

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/com.github.philip_scott.notes-up

**Comment by TingPing** [comment]
I added both you and @Philip-Scott to the repo also.

**Comment by Alexander-Wilms** [comment]
Thanks!

**Comment by Philip-Scott** [comment]
Thank you both :) 

So what do i have to do to build a new release?

**Comment by Alexander-Wilms** [comment]
You'll only need to update these two lines:

					"branch": "1.4.4",
					"commit": "8dd51ced91b0af97ef451160f2bb043994de8f6b"

https://github.com/flathub/com.github.philip_scott.notes-up/blob/master/com.github.philip_scott.notes-up.json#L99-L100

**Pull request closed without merging**